src/blocks/highlight/test/highlight.cypress.js

```diff
@@ -39,7 +39,9 @@ describe( 'Block: Highlight', function() {
 	/**
 	 * Test the accordion block content font settings
 	 */
-	it( 'Test highlight block font size setting.', function() {
+	// Disabled because fontSize controls are hidden in 5.8. This should be fixed when we can fix the fontSize controls.
+	// eslint-disable-next-line jest/no-disabled-tests
+	it.skip( 'Test highlight block font size setting.', function() {
```

Comment:
should you remove the `skip` here, now that you have a fix?

Reply:
Good call. Just pushed up some fixes to that test.